scripts/install-tree-sitter-lib

```diff
@@ -9,24 +9,23 @@ error() {
   exit 1
 }
 
-config=config.sh
-[[ -f "$config" ]] || error "Missing config file. Did you run ./configure?"
-. "$config"
-
+echo "Installing tree-sitter library directly from git."
+if [[ ! -e tree-sitter ]]; then
+  git clone --depth 1 https://github.com/tree-sitter/tree-sitter.git
```

Comment:
maybe we want to pin a specific version? just to avoid problems if tree-sitter.git gets incompatible changes.

Reply:
good idea, I'll pin tree-sitter (after merging this)